### PR TITLE
Fix share controller tint, remove empty screen for group details

### PR DESCRIPTION
**Pull request by daehn**
## What's new in this PR?

* Fix share controller tint.
* Remove empty screen for group details.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation Options/TintCorrectedActivityController.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+
+// Subclass intended to work around https://stackoverflow.com/questions/25795065/ios-8-uiactivityviewcontroller-and-uialertcontroller-button-text-color-uses-wind
+final class TintCorrectedActivityViewController: UIActivityViewController {
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Conversation Options/TintCorrectedActivityController.swift]: This can't be solved by UIApperance?

Comment (typfel) [reply Wire-iOS/Sources/UserInterface/Conversation Options/TintCorrectedActivityController.swift]: Ah read the stackoverflow link :)

File: Wire-iOS/Sources/UserInterface/MainController/ZClientViewController.m [hunk 2]
```diff
@@ -131,6 +131,7 @@ - (instancetype)init
         [[SectionHeader appearanceWhenContainedInInstancesOfClasses:@[StartUIView.class]] setColorSchemeVariant:ColorSchemeVariantDark];
         [[GroupConversationCell appearanceWhenContainedInInstancesOfClasses:@[StartUIView.class]] setColorSchemeVariant:ColorSchemeVariantDark];
         [[GroupConversationCell appearanceWhenContainedInInstancesOfClasses:@[StartUIView.class]] setContentBackgroundColor:UIColor.clearColor];
+        [[UIView appearanceWhenContainedInInstancesOfClasses:@[UIAlertController.class]] setTintColor:[ColorScheme.defaultColorScheme colorWithName:ColorSchemeColorTextForeground variant:ColorSchemeVariantLight]];
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/MainController/ZClientViewController.m]: Is this redundant then?

File: Wire-iOS/Sources/UserInterface/Conversation Options/TintCorrectedActivityController.swift (hunk 1 above)
Comment (daehn) [reply Wire-iOS/Sources/UserInterface/Conversation Options/TintCorrectedActivityController.swift]: No, I tried but it get's overwritten by the UIWindows tintColor unfortunately (see the linked SO thread).

File: Wire-iOS/Sources/UserInterface/MainController/ZClientViewController.m (hunk 2 above)
Comment (daehn) [reply Wire-iOS/Sources/UserInterface/MainController/ZClientViewController.m]: This is for UIAlertController and was there before, I only moved it to the other appearance calls from further down in the same file.

**Pull request merged**